<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="loadButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Load file</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save file</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addRowButton">
        <property name="text">
         <string>Add row</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete row</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLineEdit" name="chooseRestaurant">
        <property name="placeholderText">
         <string>Name of restaurant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="chooseRating">
        <property name="placeholderText">
         <string>Rating</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="chooseCuisine">
        <property name="placeholderText">
         <string>Cuisine</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Cost">
        <property name="placeholderText">
         <string>Cost</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="chooseFairRating">
        <property name="placeholderText">
         <string>Fair rating</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLineEdit" name="chooseVotes">
        <property name="placeholderText">
         <string>Number of votes</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="chooseAdress">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Adress</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="chooseOpenHours">
        <property name="placeholderText">
         <string>Opening hours</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="tableView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
